exporters/geneva/example/example_metrics.cc

```diff
@@ -0,0 +1,133 @@
+  std::string observable_counter_name = name + "_observable_counter";
+  std::unique_ptr<metric_sdk::InstrumentSelector>
+      observable_instrument_selector{new metric_sdk::InstrumentSelector(
+          metric_sdk::InstrumentType::kObservableCounter,
+          observable_counter_name)};
+  std::unique_ptr<metric_sdk::MeterSelector> observable_meter_selector{
+      new metric_sdk::MeterSelector(name, version, schema)};
+  std::unique_ptr<metric_sdk::View> observable_sum_view{new metric_sdk::View{
+      name, "description", metric_sdk::AggregationType::kSum}};
+  p->AddView(std::move(observable_instrument_selector),
+             std::move(observable_meter_selector),
+             std::move(observable_sum_view));
+
+  // histogram view
+  std::string histogram_name = name + "_histogram";
+  std::unique_ptr<metric_sdk::InstrumentSelector> histogram_instrument_selector{
+      new metric_sdk::InstrumentSelector(metric_sdk::InstrumentType::kHistogram,
+                                         histogram_name)};
+  std::unique_ptr<metric_sdk::MeterSelector> histogram_meter_selector{
+      new metric_sdk::MeterSelector(name, version, schema)};
+  std::shared_ptr<opentelemetry::sdk::metrics::AggregationConfig>
+      aggregation_config{
+          new opentelemetry::sdk::metrics::HistogramAggregationConfig<double>};
+  static_cast<
+      opentelemetry::sdk::metrics::HistogramAggregationConfig<double> *>(
+      aggregation_config.get())
+      ->boundaries_ = std::list<double>{0.0,   50.0,   100.0,  250.0,  500.0,
+                                        750.0, 1000.0, 2500.0, 5000.0, 10000.0};
+  std::unique_ptr<metric_sdk::View> histogram_view{new metric_sdk::View{
+      name, "description", metric_sdk::AggregationType::kHistogram,
+      aggregation_config}};
+  p->AddView(std::move(histogram_instrument_selector),
+             std::move(histogram_meter_selector), std::move(histogram_view));
+  metrics_api::Provider::SetMeterProvider(provider);
+}
+} // namespace
+
+int main(int argc, char **argv) {
+  std::string account_name = "TestAccount";
+  std::string example_type;
+  if (argc >= 2) {
+    account_name = argv[1];
+    if (argc >= 3) {
+      example_type = argv[2];
+    }
+  }
+
+  std::string name{"ostream_metric_example"};
+  initMetrics(name, account_name);
+
+  if (example_type == "counter") {
+    foo_library::counter_example(name);
+  } else if (example_type == "observable_counter") {
+    foo_library::observable_counter_example(name);
+  } else if (example_type == "histogram") {
+    foo_library::histogram_example(name);
+  } else {
+    std::thread counter_example{&foo_library::counter_example, name};
+    std::thread observable_counter_example{
+        &foo_library::observable_counter_example, name};
+    std::thread histogram_example{&foo_library::histogram_example, name};
+
+    counter_example.join();
+    observable_counter_example.join();
+    histogram_example.join();
+  }
+}
+#else
+int main() {}
```

Comment:
Better to not built an empty example in this case?

Reply:
yeah. we will remove that as part of old metrics cleanup ?